.goreleaser.yaml

```diff
@@ -1,3 +1,4 @@
+version: 2
```

Comment:
:memo: GoReleaser v2 requires the version 2 config format.

> It should only warn you about the version header in the configuration file, which you can fix by adding version: 2 to it.
https://goreleaser.com/blog/goreleaser-v2/